pom.xml

```diff
@@ -274,6 +274,11 @@
         <version>${project.version}</version>
         <scope>test</scope>
       </dependency>
+    <dependency>
+        <groupId>com.esotericsoftware</groupId>
```

Comment:
Note that this isn't actually affecting the Kryo version, since the `dev/dependencies` files weren't updated.  The motivation for this was that `unsafe` was depending on `chill`, a Scala dependency, when in fact it only needed Kryo. To remove this Chill dependency so that `unsafe` could be a pure-Java module, I replaced it by a direct dependency on Kryo and used the exact same dependency and version that Chill was pulling in.